page-1.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>National Park Adventure Guide</title>
+  </head>
+  <body>
+    <header>
+      <h1>National Park Adventure Guide</h1>
+    </header>
+    <main>
+      <section>
+        <div>Address</div>
+        <input type="text" />
+    </section>
+    <div>national parks</div>
```

Comment:
This should be inside of a section and the div should be an element. Element and aside should be the children of the section

Reply:
article, not element